### ISSUE TITLE
Upload QR code to the science portal?

### ISSUE THREAD

**Issue by bregeon** [astrolabsoftware/fink-science-portal#380]
Since you guys seem to love QR code, I was wondering if there could be a way to upload the image of a QR code on the science portal... so that one does not have to use a phone to get the object id and then get to the object page when using a laptop.

**Comment by JulienPeloton** [comment]
Good idea, thanks @bregeon ! I will see how to do this efficiently, and ask you for feedback :-)
